mlflow/server/handlers.py

```diff
@@ -262,6 +262,44 @@ def wrapper(*args, **kwargs):
 ]
 
 
+def _disable_unless_serving_artifacts(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        from mlflow.server import SERVE_ARTIFACTS_ENV_VAR
+
+        if not os.environ.get(SERVE_ARTIFACTS_ENV_VAR):
+            return Response(
+                (
+                    "Endpoint disabled due to the mlflow server running without "
```

Comment:
```suggestion
                    f"Endpoint: {request.url_rule} disabled due to the mlflow server running without "
```

Can we include `request.url_rule` to give a more detailed error message?

Reply:
definitely. updated tests as well